bodo/libs/_bodo_common.h

```diff
@@ -39,6 +41,17 @@
 // POP_IGNORED_COMPILER_ERROR. Otherwise the error will be disabled for the rest
 // of compilation
 #define POP_IGNORED_COMPILER_ERROR() DO_PRAGMA(GCC diagnostic pop)
+#elif defined(_MSC_VER)
+#define PUSH_IGNORED_COMPILER_ERROR(err) \
+    __pragma(warning(push));             \
+    __pragma(warning(disable : err))
```

Comment:
Does `-Waggressive-loop-optimizations` work on Windows? I'd guess this would fail for this line: https://github.com/bodo-ai/Bodo/blob/6dd17be35f79d5f9340e2e775cba9734437e5150/bodo/libs/_bodo_tdigest.cpp#L152